geode-core/src/distributedTest/java/org/apache/geode/internal/cache/tier/sockets/ClientServerMiscDUnitTest.java

```diff
@@ -770,25 +760,21 @@ public void testSystemCanBeCycledWithAnInitializedPool() throws Exception {
     prop.setProperty(LOCATORS, "");
     ds = getSystem(prop);
 
-    cache = getCache();
-    assertNotNull(cache);
+    final Cache cache2 = getCache();
```

Comment:
Why do we need a new copy of the cache here?

Reply:
The reason this was copied was to clear the warning for the finality required on being passed into a lambda.

Reply:
Renamed the ugly variable name.